library/core/src/fmt/mod.rs

```diff
@@ -347,7 +347,7 @@ impl<'a> Arguments<'a> {
     #[doc(hidden)]
     #[inline]
     #[unstable(feature = "fmt_internals", reason = "internal to format_args!", issue = "none")]
-    pub fn new_v1_formatted(
+    pub const fn new_v1_formatted(
```

Comment:
I'm a bit surprised this does not need a `rustc_const_unstable`... maybe we have that implicitly for `const` `#[unstable(` functions?

Reply:
yes